src/Sulu/Bundle/LocationBundle/DependencyInjection/Configuration.php

```diff
@@ -32,7 +32,7 @@ public function getConfigTreeBuilder()
                             ->addDefaultsIfNotSet()
                             ->children()
                                 ->scalarNode('api_key')->defaultValue('')->end()
-                                ->scalarNode('endpoint')->defaultValue('http://open.mapquestapi.com/nominatim/v1/search.php')->end()
+                                ->scalarNode('endpoint')->defaultValue('https://www.mapquestapi.com/geocoding/v1/address')->end()
```

Comment:
Should we change to the nominate endpoint suggested here by 
@remdan: https://github.com/sulu/sulu-docs/pull/781#issuecomment-1816560636

Does somebody know the history why? Not longer works? http://open.mapquestapi.com/nominatim/v1/search.php

Was the whole endpoint moved to `https://nominatim.openstreetmap.org/search?`?